packages/cli/src/lib/yargs-cli.ts

```diff
@@ -90,3 +94,15 @@ export function yargsCli<T = unknown>(
   // return CLI object
   return cli as unknown as Argv<T>;
 }
+
+function validatePersistFormat(persist: PersistConfig) {
+  try {
+    return (
+      persist?.format == null ||
+      persist.format.map(format => formatSchema.parse(format)).length ===
+        persist.format.length
+    );
```

Comment:
This is written in an unintuitive way, `arr.map(...).length === arr.length` is a logical tautology. The point is that none of the `formatSchema.parse` calls through, right? In that case, use `forEach`, that's what's intended for side-effects.

```suggestion
    persist.format?.forEach(format => {
      formatSchema.parse(format);
    });
    return true;
```